index.src.html

```diff
@@ -61,8 +61,8 @@
 ## Registration (<a>embedded authenticator</a> mode) ## {#registration-embedded}
 
 - On the phone:
-    * User goes to example.com in the browser, and signs in using whatever method they have been using (possibly a legacy
-        method such as a password).
+    * User goes to example.com in the browser and signs in to an existing account using whatever method they have been using
```

Comment:
I nominally agree with @leshi   -- further, i suggest choosing between these two possible resolutions: 
a) use "sign-in"/"signs-in" (or "sign-on"/"signs-on") consistently in spec rather than "sign in"/"signs in", and then this sentence has "signs-in to"
b) use "signs into" here

fwiw, I prefer (a).